lib/script/script.js

```diff
@@ -1312,7 +1312,7 @@ class Script {
           // Make sure the split point is appropriate.
           const pos = stack.getInt(-1, minimal, 4);
           if (pos < 0 || pos > data.length)
-            throw new ScriptError('INVALID_SPLIT_RANGE', op, ip);
```

Comment:
I would keep the prefix for these error codes. `INVALID` provides context to the error and is also consistent with other script error codes e.g: `INVALID_STACK_OPERATION`.

Reply:
Let's ask ABC folks upstream to make this change so we don't have to keep doing this.